ui/app/styles/components/console-ui-panel.scss

```diff
@@ -118,11 +120,11 @@ $console-close-height: 35px;
 
 .panel-open .console-ui-panel {
   box-shadow: $box-shadow-highest;
-  min-height: 400px;
+  min-height: 425px;
```

Comment:
I think eventually we'll want to re-visit this to make it a little more flexible, but right now it needed to be increased to account for the close button. 

The close button was positioned in the DOM outside of the container it was then positioned inside of, which was causing the element to overlap. We don't want it to overlap other content.